CLI/src/CLI.cs

```diff
@@ -524,6 +528,11 @@ private ReturnCode AddErrorsAndComputeReturnCode()
             CheckExternalDiagnostics(_dependenciesDiagnostics);
             CheckExternalDiagnostics(_intrinsicsDiagnostics);
 
+#if EUROINFO_RULES
+            if (this._configuration.ReportUsedCopyNamesPath != null && _configuration.ExecToStep <= ExecutionStep.Preprocessor)
+                return returnCode;
+#endif
```

Comment:
I don't see why we need to do that.

Reply:
To avoid return ReturnCode.MissingCopy

Reply:
Ok so this logic should also apply to `-exc` option.

What about something like:

```csharp
if (_missingCopies.Count > 0 && _configuration.ExtractedCopiesFilePath != null && _configuration.ReportUsedCopyNamesPath != null)
{
    returnCode = ReturnCode.MissingCopy;
}
```

Reply:
the thing is that -cpyr options is only an EI option, and in this situation the MissingCopy is not considered as an error, this options is intended to be called even if the collected copy names are missing, there is also another condition _configuration.ExecToStep <= ExecutionStep.Preprocessor